libs/components/src/next/form/select/vl-select.component.ts

```diff
@@ -93,19 +93,28 @@ export class VlSelectComponent extends FormControl {
     updated(changedProperties: Map<string, unknown>): void {
         super.updated(changedProperties);
 
+        if (!this.choices) {
+            return;
+        }
+
         if (changedProperties.has('options')) {
-            this.choices?.clearStore();
-            this.choices?.setChoices(this.getOptions(), 'value', 'label', true);
+            this.choices.clearStore();
+            this.choices.setChoices(this.getOptions(), 'value', 'label', true);
             this.onChange();
         }
 
         if (changedProperties.has('disabled')) {
             if (this.disabled) {
-                this.choices?.disable();
+                this.choices.disable();
             } else {
-                this.choices?.enable();
+                this.choices.enable();
             }
         }
+
+        if (changedProperties.has('deletable')) {
```

Comment:
mss te laat om op te merken, maar zou verwachten dat `deletable` standaard `true` zou staan (is ook zoals het werkt voor de multiselect by default)

Reply:
Ik heb dit standaard op false gezet omdat het een variant is, als alle afnemers dat standaard op true zetten kunnen we dat gerust aanpassen.

Reply:
Ok - that tracks.
Zou te gek zijn om default te veranderen als die op `multiselect` komt te staan.